src/eve-server/system/DestinyManager.cpp

```diff
@@ -3139,31 +3330,58 @@ void DestinyManager::SendDestinyUpdate( std::vector<PyTuple*>& updates, std::vec
-                    (mySE->HasPilot()?mySE->GetPilot()->GetName():mySE->GetName()),\
-                    (mySE->HasPilot()?mySE->GetPilot()->GetCharID():mySE->GetID()) );
+        if (is_log_enabled(DESTINY__UPDATES)) {
+            _log(
+                DESTINY__UPDATES,
+                "[%u] BubbleCasting global structure destiny update (u:%u, e:%u) for stamp %u to all bubbles from %s(%u)",
+                sEntityList.GetStamp(),
+                updates.size(),
+                events.size(),
+                sEntityList.GetStamp(),
+                (mySE->HasPilot()?mySE->GetPilot()->GetName():mySE->GetName()),
+                (mySE->HasPilot()?mySE->GetPilot()->GetCharID():mySE->GetID())
+            );
+        }
 
         //Get all clients in the system which the SE is in
         /** @todo  this isnt right....will segfault.  needs to be fixed */
         std::vector<Client*> cv;
+
         mySE->SystemMgr()->GetClientList(cv);
+
         for(auto const& value: cv) {
             value->GetShipSE()->SysBubble()->BubblecastDestiny(updates, events, "destiny");
         }
     } else if (mySE->SysBubble() != nullptr) {
-        if (is_log_enabled(DESTINY__UPDATES))
-            _log(DESTINY__UPDATES, "[%u] BubbleCasting destiny update (u:%u, e:%u) for stamp %u to bubbleID %u from %s(%u)", \
-                    sEntityList.GetStamp(), updates.size(), events.size(), sEntityList.GetStamp(), mySE->SysBubble()->GetID(),   \
-                    (mySE->HasPilot()?mySE->GetPilot()->GetName():mySE->GetName()),\
-                    (mySE->HasPilot()?mySE->GetPilot()->GetCharID():mySE->GetID()) );
+        if (is_log_enabled(DESTINY__UPDATES)) {
+            _log(
+                DESTINY__UPDATES,
+                "[%u] BubbleCasting destiny update (u:%u, e:%u) for stamp %u to bubbleID %u from %s(%u)",
+                sEntityList.GetStamp(),
+                updates.size(),
+                events.size(),
+                sEntityList.GetStamp(),
+                mySE->SysBubble()->GetID(),
+                (mySE->HasPilot()?mySE->GetPilot()->GetName():mySE->GetName()),
+                (mySE->HasPilot()?mySE->GetPilot()->GetCharID():mySE->GetID())
+            );
+        }
+
         mySE->SysBubble()->BubblecastDestiny( updates, events, "destiny" );
     } else {
-        _log(DESTINY__ERROR, "[%u] Cannot BubbleCast destiny update (u:%u, e:%u); entity (%u) is not in any bubble.", \
-                sEntityList.GetStamp(), updates.size(), events.size(), mySE->GetID() );
-        if (sConfig.debug.IsTestServer)
-            EvE::traceStack();
+        _log(
+            DESTINY__ERROR,
+            "[%u] Cannot BubbleCast destiny update (u:%u, e:%u); entity (%u) is not in any bubble.",
+            sEntityList.GetStamp(),
+            updates.size(),
+            events.size(),
+            mySE->GetID()
+        );
+
+        // if (sConfig.debug.IsTestServer) {
+        //     EvE::traceStack();
+        // }
+
```

Comment:
more simple code formatting fixes, but also removes the traceStack call because it isn't really necessary for this situation (based on my experience debugging destiny).